src/snapshotter.js

```diff
@@ -133,6 +135,39 @@ export default class SnapShotter {
     }
   }
 
+  switchToIFrame(selector, timeout) {
+    return this.driver.wait(
+      this._until.ableToSwitchToFrame(this._By.css(selector)),
+      timeout
+    );
+  }
+
+  waitToLocateElement(selector, timeout) {
+    return this.driver.wait(
+      this._until.elementLocated(this._By.css(selector)),
+      timeout
+    );
+  }
+
+  async waitForIFrameElement() {
+    const timeout = 10000;
+
+    const { frame, element } = this._waitForIFrameElement;
+
+    try {
+      await this.switchToIFrame(frame, timeout);
+      await this.waitToLocateElement(element, timeout);
+    } catch (error) {
+      console.log(''); // eslint-disable-line no-console // space for progress bar
```

Comment:
This line is taken from the waitForElement function. To help with an issue with the updating progress bar.